README.md

```diff
@@ -1,5 +1,49 @@
-# injected
-Source code of DLLs / assemblies for injection into target process
+# injectdll
 
-usage:
-py -2.7 example_injector.py \<Process name\> \<path to dll\>
+injectdll is a Python package that allows you to inject DLLs into running processes on Windows (32 or 64-bit).
+
+This can be useful for various tasks, such as debugging or adding functionality to existing applications.
```

Comment:
-> `This is initially created for desktop GUI automation purpose (project pywinauto), because an injected DLL can access GUI widgets' text properties with much better precision and coverage than standard OS APIs like MS UI Automation API or Win32 API. Other use cases are potentially possible at your own risk according to the law and the project license.`